src/components/Sticky.vue

```diff
@@ -82,8 +86,8 @@ export default {
     mouseDown(e) {
       e.cancelBubble = true
       this.dragging = true
-	  this.layerPos.x = this.$el.offsetWidth / 2 - e.pageX + this.$el.offsetLeft
-	  this.layerPos.y = this.$el.offsetHeight / 2 - e.pageY + this.$el.offsetTop
+      this.layerPos.x = this.$el.offsetWidth / 2 - e.pageX + this.$el.offsetLeft
+      this.layerPos.y = this.$el.offsetHeight / 2 - e.pageY + this.$el.offsetTop
```

Comment:
sorry, yesterday pull request included tab that was merged master.
this commit is replace tab to space